src/test/scala/io/iohk/ethereum/vm/CreateOpcodeSpec.scala

```diff
@@ -248,6 +248,13 @@ class CreateOpcodeSpec extends AnyWordSpec with Matchers with ScalaCheckProperty
       "leave return buffer empty" in {
         result.stateOut.returnData shouldEqual ByteString.empty
       }
+
+      "add the new contract to accessed_addresses" in {
+        result.world.getGuaranteedAccount(newAccountAddress())
+        val addr = newAccountAddress()
```

Comment:
I would have called result.world.getGuaranteedAccount with the val addr instead re-calling newAccountAddress().
For create2, this seems to not be significant, but for create, there is a slight side effect with the nonce being modified.

Unless there is a good reason to call multiple time newAccountAddress()

```
val addr = newAccountAddress()
result.world.getGuaranteedAccount(addr)
result.stateOut.accessedAddresses should contain(addr)

```